changelog.d/20250430_152200_8730430+m1yag1_sc_40419_filter_roles_list_flows_and_runs.md

```diff
@@ -0,0 +1,7 @@
+
+### Enhancements
+
+* Updated `--filter-role` parameter in  `globus flows list` to be
+  supplied multiple times to specify multiple roles.
+* Added `--filter-role` parameter to `globus flows run list` which can
+  be supplied multiple times to specify multiple roles.
```

Comment:
Minor, but for a new flag I'd tend to reduce the level of detail in the changelog so that it stands out better.

```suggestion
* Added `--filter-role` parameter to `globus flows run list`.
```

I think in terms of how I read changelogs when I'm trying to figure out when a feature was added.